### DISCUSSION TITLE
Making progress with some major things

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#70] [Announcements]
I figured out how to do particles a bit. Just gotta refine it later, but here's some of my work. Enjoy!

![2022-06-14_19 25 18](https://user-images.githubusercontent.com/50820746/173727699-72a9587c-a60b-4c61-9a07-f8a8b91d68ab.png)

![2022-06-14_19 30 24](https://user-images.githubusercontent.com/50820746/173727783-cac3239a-c522-482b-8a8f-73c9a19944e5.png)

![2022-06-14_19 31 15](https://user-images.githubusercontent.com/50820746/173727805-a81fa6eb-aa83-4fd1-9f6f-7e6727f2e9c2.png)

![2022-06-14_19 36 13](https://user-images.githubusercontent.com/50820746/173727822-bb760b2b-6cd7-4ab9-b180-3558cb2b3685.png)

![2022-06-14_19 48 24](https://user-images.githubusercontent.com/50820746/173727843-f505b75c-3bd4-409c-ba6d-eb8a99a61271.png)

![2022-06-14_19 52 07](https://user-images.githubusercontent.com/50820746/173727870-29554fb9-09f4-437e-b9d6-31523a2fb96c.png)

![2022-06-14_19 36 09](https://user-images.githubusercontent.com/50820746/173727903-a7f5d82e-c375-48c7-a26e-bdc811a46978.png)

**Comment by Mrbt0907** [comment]
![2022-06-14_19 36 33](https://user-images.githubusercontent.com/50820746/173728012-b20fd8b2-dcc1-4e91-80cc-d7623f9f7028.png)

Forgot one

**Reply by Joschac2** [reply]
Will storms make days darker and nights even more darker?

**Reply by Mrbt0907** [reply]
Currently, they use vanilla's mechanics to make the sky darker just like rain, and I am not sure about night time.

**Comment by odvigajlo** [comment]
I'm soooo excited for this update

**Comment by Jaytv25** [comment]
How many requests are in that update exactly?

**Reply by Mrbt0907** [reply]
14 Requests and 2 Pages of requests is what will be in this update once it's finished.

Once I finish the new storm system, I will release an alpha for 2.8

**Comment by Joschac2** [comment]
Did yoy change how snow works.   It used to snow in the winter time with Serene Seasons. But now with this version, it just rains?

**Reply by Mrbt0907** [reply]
The original Serene Season compatibility does not work right now. I gotta fix that in the next update
